ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/DefaultNamingStrategy.java

```diff
@@ -19,12 +20,18 @@
  *
  * @author Mathieu Bague <mathieu.bague@rte-france.com>
  */
+@AutoService(NamingStrategy.class)
 public class DefaultNamingStrategy implements NamingStrategy {
 
     private final Map<String, UcteNodeCode> ucteNodeIds = new HashMap<>();
 
     private final Map<String, UcteElementId> ucteElementIds = new HashMap<>();
 
+    @Override
+    public String getName() {
+        return "DefaultNamingStrategy";
```

Comment:
Change to « Default » to write something like:
ucte.exporter.naming-strategy=Default